eng/pipelines/git-tasks.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  - name: TaskType
+    displayName: Task Task
+    type: string
+    default: DeleteTag
```

Comment:
Did you want to limit this to just tags? Should it be for branches as well?

Reply:
We can expand from here. Branches can be easily deleted from the UI whereas deleting a remote tag requires a git command.